src/defaults.js

```diff
@@ -7,12 +7,36 @@ const { defaultOptions } = require("./utils");
  */
 const TUS_CHUNK_SIZE = (1 << 22) * 10;
 
+/**
+ * Indicates what the default chunk size multiplier is.
+ */
+const DEFAULT_TUS_CHUNK_SIZE_MULTIPLIER = 1;
+
+/**
+ * Indicates how many parts should be uploaded in parallel, by default.
+ */
+const DEFAULT_TUS_PARALLEL_UPLOADS = 1;
```

Comment:
same comment as above

https://github.com/SkynetLabs/skynet-js/blob/master/src/upload.ts#L31

```suggestion
const DEFAULT_TUS_PARALLEL_UPLOADS = 2;
```

Reply:
Hello **@kwypchlo,**

I know that the constants in "SkynetLabs/skynet-js" are set differently.

But the problem is that if I set the constant like this:
`const DEFAULT_TUS_PARALLEL_UPLOADS = 2;`
then "parallelUploads" is always executed for each large file upload, even if the "numParallelUploads" option is not set for "uploadFile".

So we should set it like this:
`const DEFAULT_TUS_PARALLEL_UPLOADS = 1;`
so that without the "numParallelUploads" option set, a large file upload also works without "parallelUploads".
If option "{ numParallelUploads = 2 }" is set, then "parallelUploads" will be executed as intended.

Reply:
Hello @kwypchlo,

What should I do?
Change to the constants from "SkynetLabs/skynet-js" or rather leave both constants at 1 for now, because it works great and there were no errors.

let me know

Reply:
those defaults were set deliberately for best upload performance out of the box, let's copy them from skynet-js please